apps/mobile/src/styles/styles.ts

```diff
@@ -0,0 +1 @@
+export { globalStyles } from "./global-styles/global-styles";
```

Comment:
You have a linter error about unused files, so just use one of the styles (globalStyles.flexDirectionRow, for example) somewhere.